modules/custom/az_migration/config/install/migrate_plus.migration.az_node_person.yml

```diff
@@ -0,0 +1,150 @@
+id: az_node_person
+label: Person Node Content
+migration_group: az_migration
+status: true
+
+source:
+  plugin: az_person
```

Comment:
@smitahantzios Did you try using just the standard `d7_node` source plugin here rather than implementing your own?  There is nothing too special about the D7 Person content type so I was pretty surprised to see all of the code in your custom Person source plugin.  I don't really think all of that custom code should be necessary to get the source data from the D7 person content type.

Reply:
(Something like this?)

```source:
  plugin: d7_node
  node_type: uaqs_person
```

Reply:
Agreed - I think it'd be desirable to use the standard `d7_` plugins where possible.

Reply:
D7 person content type - 15 fields. Out of 15 only two having multi value fields. So while fetching and mapping need to loop only for this two fields, to make it more faster migration process. "d7_node" plugin use function "getFieldValues" - also executes the same database query in generic way and fetch all values. To make the migration process more faster in large number of records, wrote the custom Plugin. Please let me know your thoughts. Thanks.

Reply:
In my opinion using the core `d7_node` plugin is preferable to adding a custom `az_person` plugin from the standpoint of code maintainability. Additional speed is a good feature to have, but migrations will generally be run only once per site.